src/actions.js

```diff
@@ -32,6 +40,11 @@ export function fetchBatchRunSummaries(mm, filters) {
   return graphql(payload, 'CLAIM_BATCH_CLAIM_BATCH_SEARCHER');
 }
 
+export function fetchBatchRunReadOnlyPicker(params) {
+  const payload = formatPageQueryWithCount("batchRuns", params, BATCH_RUN_READ_ONLY_PICKER_PROJECTION);
```

Comment:
Do you need to use `formatPageQueryWithCount`? Why not `formatPageQuery`? These options are fetched for the picker, not to be displayed in a table with pagination, correct?